docs/src/feather/getting-started.md

```diff
@@ -0,0 +1,60 @@
+# Plugins
+
+## Install
+### To install Feather, if you are running Unix,
+run the following in your terminal, then follow the onscreen instructions.
+```sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.feathermc.org | sh
+```
+
+### If you are running Windows 64-bit,
+download and run \
+[feather-init.exe](https://win.feathermc.org/x86_64) \
+then follow the onscreen instructions.
+
+### If you are running Windows 32-bit,
+download and run \
+[feather-init.exe](https://win.feathermc.org/i686) \
+then follow the onscreen instructions. 
+
+
+## Commands
+### Init
+To create a server in an exisiting directory
+```sh
+feather init [path]
```

Comment:
We might want to rename the CLI tool to `feather server`, since in the long term we may implement a `feather-client.`

Reply:
Hmm should a feather-client maybe have its own name, like quill has its own name?